src/resolvers/variable.js

```diff
@@ -4,6 +4,39 @@ import { handlePagingParams } from '../utils/handlePagingParams'
 import { isDraftConceptId } from '../utils/isDraftConceptId'
 
 export default {
+  Mutation: {
+    publishGeneratedVariables: async (source, args, context, info) => {
+      const { conceptId } = args
+
+      const { dataSources } = context
+
+      const results = await dataSources.collectionVariableDraftsSource({
+        conceptId,
+        publish: true
+      }, context, parseResolveInfo)
+
+      // Pull out the variable concept ids from the source to use as parameters later
+      const conceptIds = {
+        params: { conceptId: Object.values(results).map((object) => Object.values(object)).flat() }
```

Comment:
Can we split this up a bit or double check these all calls need to be made here?